sidebars.ts

```diff
@@ -186,8 +186,9 @@ module.exports = {
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloud-to-cloud-source-versions',
+                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-fdr-host-inventory-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-fdr-source',
-                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-source',
+                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-source',          
```

Comment:
Have you created this file? "crowdstrike-source"

Reply:
No. It already exists

Reply:
But this file has this change.

Reply:
Yes, it had an extra tab there. I have removed it now